src/components/admin/services/DropDown.jsx

```diff
@@ -51,7 +51,7 @@ const Menu = ({ setOption, className, setOptions, options }) => {
             </Dropdown.Item>
           ))
       ) : (
-        <p className="px-3 text-gray-400">no events</p>
+        <p className="px-3 text-gray-400"></p>
```

Comment:
this is the text displace when there's no options, so it should be a customized text pass in dropdown as a prop